Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -1175,6 +1176,22 @@ void CFrame::OnNetPlay(wxCommandEvent& WXUNUSED(event))
   }
 }
 
+void CFrame::OnLua(wxCommandEvent& WXUNUSED(event))
+{
+
+  if (!m_lua_script_frame)
+  {
+    m_lua_script_frame = new Lua::LuaScriptFrame(this);
+
+    Movie::SetGCInputManip([this](GCPadStatus* pad, int controller_id) {
+      if (m_lua_script_frame)
+        m_lua_script_frame->GetValues(pad);
+    }, Movie::GCManipIndex::LuaGCManip);
+  }
+  else
+    m_lua_script_frame->Raise();
```

Comment:
This should be braced, since the if-portion is braced, like what our coding style says to do.